webapp/src/components/AuctionPage/Token/Token.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Header } from 'semantic-ui-react'
+
+import './Token.css'
+
+export default class Token extends React.PureComponent {
+  static propTypes = {
+    symbol: PropTypes.string.isRequired,
+    amount: PropTypes.number.isRequired,
+    loading: PropTypes.bool
+  }
+  render() {
+    const { symbol, amount, loading } = this.props
+    if (loading) {
+      return (
+        <Header size="large" className="Token">
+          <span className="loading">--.--</span> {symbol}
+        </Header>
+      )
+    }
+    return (
+      <Header size="large" className="Token">
+        {symbol === 'MANA' ? (
+          <span>⏣ {amount}</span>
```

Comment:
This Mana looks awesome but isn't it confusing to have it just here? the rest of the site uses the old one

Reply:
Gotcha